doc/adr/0013-marconi-query-interface.rst

```diff
@@ -0,0 +1,130 @@
+.. _marconi_indexer:
+
+ADR 13: Marconi Query Interface
+======================================================
+
+Date: 2022-10-27
+
+Author(s)
+---------
+
+Kayvan Kazeminejad <kayvan.kazeminejad@iohk.io>
+
+Status
+------
+
+Draft
+
+Context
+-------
+Marconi Query Interface is both an executable and a Haskell library built on top of `marconi`.  In its current alpha version, the interface provides a reporting capability for a bounded list of `Bech32 Shelley Era addresses <https://github.com/cardano-foundation/CIPs/tree/master/CIP-0019#shelley-addresses>`_ and their corresponding `UTXO` `TxOutRef`.  The executable package provides for an HTTP `JSON-RPC 2.0 <https://www.jsonrpc.org/specification>`_ on top of the marconi Haskell libraries.
+
+
+Scope
+-------
+
+The scope of this document is the marcoin query interface executable package built on top:
+
+- `marconi`
+- `marconi-query`
+- JSON-RPC 2.0
+- HTTP
+
+Caution
+^^^^^^
+
+This is a living document as the interface is under development
+
+
+Decision
+--------
+
+After soliciting a number of potential clients, we made the following architectural decisions:
+
+* Minimize changes to the marconi back-end
+* JSON-RPC 2.0 for non-Haskell users
+* Bech32 Shelley Era addresses are provided through CLI
+* The alpha release will scope to reporting on `TxOutRef` of `UTXO`s only
+
+These decision were an architectural compromise with the following  cons and pros:
```

Comment:
I suggest making "decision" plural and you may as well say "pros and cons" in that order since that's the order you're using below. 

"These decisions were an architectural compromise with the following pros and cons:"